comps/embeddings/deployment/kubernetes/helm-chart/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+name: embedding-usvc
+description: The Helm chart for deploying embedding as microservice
+type: application
+version: 0-latest
+# The embedding microservice server version
+appVersion: "v1.0"
+dependencies:
+  - name: tei
+    version: 0-latest
+    repository: file://../../../../3rd_parties/tei/deployment/kubernetes/helm-chart
```

Comment:
should this point to an internal OCI repository?

Reply:
yes, oci references. That way the user can consume the charts without the git tree.

Reply:
We shouldn't point this to internal OCI repo.(This can be used by external users who doesn't have access to internal CI)
The Option is to use ghcr.io/opea-project/charts, but CI will need to handle more things to test latest changes, and avoid polluting the 0-latest stuff:
1. Maintain an internal registry.
2. Push the changed helm chart to internal registry, with version number indicating the tested PR. For example 0-hash
3. Replace the ghcr.io reference with internal registry at CI for the chart under test.
4. Replace the version: 0-latest with version: 0-hash
5. Cleanup the new pushed helm charts.

Ideally, for docker images we should use the same steps.

Using file:// within the same repo is a workaround to avoid such complex changes.
I can do this but it involve more CI infra setups and takes more time.
Let's finalized other parts first before doing this changes.

Reply:
What is internal OCI repo? I only know of `oci://ghcr.io/opea-project/charts` where we host the charts, and that is public.